assets/js/googlesitekit-base.js

```diff
@@ -26,6 +26,9 @@ import {
 	clearWebStorage,
 } from './util/standalone';
 
+// Load image for the parent menu item.
+import '../images/logo-g_white_small.png';
```

Comment:
It's used in PHP as a menu icon: https://github.com/google/site-kit-wp/blob/3571133f387d089c50796748e918b3c50c6136b3/includes/Core/Admin/Screen.php#L126-L133

Reply:
Yeah, I saw the comment there but it wasn't immediately obvious what the import does.
Is it because the image won't exist in the `dist` directory if it isn't imported via JS?

Reply:
Yep, if we don't import it here, it won't be loaded by webpack and won't be copied to the dist folder.